Source/AttachedFileHelper.h

```diff
@@ -93,10 +91,10 @@ namespace FFmpegInteropX
         }
 
     private:
-        std::map<String^, StorageFile^> extractedFiles;
```

Comment:
Okay so std::map does not work because StorageFile does not have default ctor. Yeah I read something about that in the docs. C++/WinRT really has its pitfalls, using structs over classes everywhere.